code/game/machinery/doors/door.dm

```diff
@@ -102,6 +102,10 @@ var/list/all_doors = list()
 				playsound(src.loc, 'sound/machines/denied.ogg', 50, 1)
 				door_animate("deny")
 
+/obj/machinery/door/bumped_by_firebird(var/obj/structure/bed/chair/vehicle/wizmobile/W)
+	W.forceMove(get_step(W,W.dir))//Firebird doesn't wait for no slowpoke door to fully open before dashing through!
```

Comment:
You mean add an instant parameter where it doesn't sleep?
I'd still have to magically make the firebird noclip through bolted/welded airlocks some other way